website/react-magma-docs/src/pages/api/transition.mdx

```diff
@@ -0,0 +1,599 @@
+          style={{
+            height: '250px',
+            width: '250px',
+            backgroundColor: '#bada55',
+          }}
+        />
+      </Transition>
+      <Button onClick={() => setIsOpen(!isOpen)}>
+        {isOpen ? 'hide' : 'show'}
+      </Button>
+    </div>
+  );
+}
+```
+
+### NudgeTop
+
+```typescript
+import React from 'react';
+import { Transition } from 'react-magma-dom';
+
+export function Example() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  return (
+    <div>
+      <div style={{ height: '250px', width: '250px', overflow: 'hidden' }}>
+        <Transition in={isOpen} unmountOnExit nudgeTop>
+          <div
+            style={{
+              height: '250px',
+              width: '250px',
+              backgroundColor: '#bada55',
+            }}
+          />
+        </Transition>
+      </div>
+      <Button onClick={() => setIsOpen(!isOpen)}>
+        {isOpen ? 'hide' : 'show'}
+      </Button>
+    </div>
+  );
+}
+```
+
+### NudgeLeft
+
+```typescript
+import React from 'react';
+import { Transition } from 'react-magma-dom';
+
+export function Example() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  return (
+    <div>
+      <div style={{ height: '250px', width: '250px', overflow: 'hidden' }}>
+        <Transition in={isOpen} unmountOnExit nudgeLeft>
+          <div
+            style={{
+              height: '250px',
+              width: '250px',
+              backgroundColor: '#bada55',
+            }}
+          />
+        </Transition>
+      </div>
+      <Button onClick={() => setIsOpen(!isOpen)}>
+        {isOpen ? 'hide' : 'show'}
+      </Button>
+    </div>
+  );
+}
+```
+
+### NudgeBottom
+
+```typescript
+import React from 'react';
+import { Transition } from 'react-magma-dom';
+
+export function Example() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  return (
+    <div>
+      <div style={{ height: '250px', width: '250px', overflow: 'hidden' }}>
+        <Transition in={isOpen} nudgeBottom>
+          <div
+            style={{
+              height: '250px',
+              width: '250px',
+              backgroundColor: '#bada55',
+            }}
+          />
+        </Transition>
+      </div>
+      <Button onClick={() => setIsOpen(!isOpen)}>
+        {isOpen ? 'hide' : 'show'}
+      </Button>
+    </div>
+  );
+}
+```
+
+### NudgeRight
+
+```typescript
+import React from 'react';
+import { Transition } from 'react-magma-dom';
+
+export function Example() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  return (
+    <div>
+      <div>
+        <Transition in={isOpen} nudgeLeft>
+          <div
+            style={{
+              width: '500px',
+              height: '100%',
+              backgroundColor: '#bada55',
+            }}
+          />
+        </Transition>
+      </div>
+      <Button onClick={() => setIsOpen(!isOpen)}>
+        {isOpen ? 'hide' : 'show'}
+      </Button>
+    </div>
+  );
+}
+```
+
+### SlideTop
```

Comment:
Are the Slide transitions always based on fixed positioning?  If so, I would specify this in the docs here.